gradle.properties

```diff
@@ -4,7 +4,8 @@ org.gradle.jvmargs=-Xmx1G
 # check these on https://fabricmc.net/develop
 minecraft_base_version=1.19
 minecraft_version=1.19.2
-yarn_mappings=1.19.2+build.11
+minecraft_version_full=1.19.2
```

Comment:
Why did you create this property? `minecraft_version` should be perfectly sufficient

Reply:
Malilib would need 1.19.0 for example.

Reply:
But then you declare 1.19.2? In any case, this should be part of the Malilib declaration and not some random property with a confusing name

Reply:
Well, that's a question to masa. If naming remains the same malilib 1.20 would be named 1.20.0. It doesn't make sense to add that field yet, but its going to be needed in the future.

Reply:
Regardless, I do not want this to be a property, especially with this kind of misleading name. Please just move it into the Malilib dependency declaration

Reply:
So leave it as just `fi.dy.masa.malilib:malilib-fabric-${project.minecraft_version}:${project.malilib_version}`?

Reply:
Well no, just make it the actual full name of the artifact, including the Minecraft version